### PR TITLE
Complete the community governance

### PR DESCRIPTION
**Pull request by fisherxu**
TSC, SIGs, Sub projects, and WGs in the community have been operating for several years in the past, and the related governance is completed in this PR.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -30,19 +30,62 @@ See [community membership](https://github.com/kubeedge/community/blob/master/com
...
+specific topic. SIGs are **permanent** until dissolved by TSC.
 
 The goal is to enable a distributed decision structure and code ownership,
 as well as providing focused forums for getting work done, making decisions,
 and on-boarding new Contributors. Every identifiable part of the project
 (e.g. repository, subdirectory, API, test, issue, PR) is intended to be
 owned by some SIG.
 
+SIGs must have open membership and always operate **in the open**.The leadership(SIG chairs and technical leads) 
+changes of SIGs is decided by the technical steering committee, for bounded or unbounded duration. And all
```
Comment (kevin-wangzefeng) [inline GOVERNANCE.md:87]:
```suggestion
changes of SIGs need approval by the technical steering committee, for bounded or unbounded duration. And all 
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -79,9 +123,72 @@ summary of the SIG's work to the community. SIGs are also responsible to:
...
   to private emails and meetings.
 
+#### Technical Leads
+
+Technical Leads are responsible for leading the SIG in correspondence with its technical 
+alignment. This alignment includes both internal to the SIG and, more significantly, external
+to the entire project.
+
+SIGs can decide on their own if they want to add Technical Leads to their charter or not. 
+Depending on the overall size of the SIG, around two to three people can be chosen by the
```
Comment (kevin-wangzefeng) [inline GOVERNANCE.md:133]:
```suggestion
Depending on the overall size of the SIG, around two to three people can be nominated by the 
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -79,9 +123,72 @@ summary of the SIG's work to the community. SIGs are also responsible to:
...
+overlapping) sets of reviewers and approvers, who act as
+subproject’s technical leaders: responsible for vision and direction
+and overall design, choose/approve project approvers/reviewers,
+field technical escalations, etc.
+
+
+### Working Groups (WGs)
+
+Working Groups provide a formal avenue for disparate groups to collaborate around a common problem, 
+craft a balanced position, and **dissolved** after the problem solved by TSC. Because they represent the
```
Comment (kevin-wangzefeng) [inline GOVERNANCE.md:152]:
```suggestion
craft a balanced position, and **dissolved** by TSC after the problem solved. Because they represent the
```

**Review by kevin-wangzefeng — COMMENTED** [review]
Minor comments, thanks.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -51,6 +94,7 @@ time. SIG chairs are intended to be organizers and facilitators, responsible for
 the operation of the SIG and for communication and coordination with the other
 SIGs, and the broader community.
 
+
```
Comment (fujitatomoya) [inline GOVERNANCE.md:97]: unnecessary change?

File: GOVERNANCE.md (hunk 3 above)
Comment (fisherxu) [reply GOVERNANCE.md]: Done.

File: GOVERNANCE.md (hunk 4 above)
Comment (fisherxu) [reply GOVERNANCE.md]: Done :) PTAL

**Review by fujitatomoya — APPROVED** [review]
lgtm

**Review by dingyin — COMMENTED** [review]
/lgtm

**Comment by kevin-wangzefeng** [comment]
We've got 5/7 approval from TSC members (including @fisherxu as PR submitter), thank you all for the support.
/approve
